src/components/Nav/Nav.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './Nav.css';
+
+function Nav() {
```

Comment:
Use full name for components
```suggestion
function Navigation() {
```